docs/source/release_notes.rst

```diff
@@ -6,15 +6,28 @@ Release Notes
 Future Release
 ==============
     * Enhancements
+        * Pass additional progress information in callback functions (:pr:`979`)
     * Fixes
     * Changes
     * Documentation Changes
     * Testing Changes
         * Add env setting to update checker (:pr:`978`)
 
     Thanks to the following people for contributing to this release:
-    :user:`gsheni`
+    :user:`gsheni`, :user:`thehomebrewnerd`
+
+Breaking Changes
+++++++++++++++++
+    * Progress callback functions are now expected to accept five parameters: 
     
+        * progress increment since last call
+        * progress units complete so far
+        * total units to complete
+        * the progress unit of measurement
+        * time elapsed since start of calculation
+        
+    Additionally, progress is now being reported in the units specified by the unit of measurement parameter instead of percentage of total.
```

Comment:
https://woodwork.alteryx.com/en/callback-update/release_notes.html

This indentation looks off

Reply:
I reworked the entire block to (hopefully) better adhere to the bulleted format we have been using, combining the leading and trailing sentences into a single first-level bullet.

Reply:
looks good!